support-frontend/assets/pages/contributions-landing/components/ContributionFormFields.jsx

```diff
@@ -163,6 +163,4 @@ function FormFields(props: PropTypes) {
   );
 }
 
-const ContributionFormFields = connect(mapStateToProps, mapDispatchToProps)(FormFields);
-
-export { ContributionFormFields };
+export default connect(mapStateToProps, mapDispatchToProps)(ContributionFormFields);
```

Comment:
🥇 for bringing this in to line with the repo-wide convention that components have default exports

Reply:
👍